simphony/testing/tests/test_utils.py

```diff
@@ -356,6 +841,18 @@ def test_create_particles(self):
                 particle.data, create_data_container(constant=index),
                 testcase=self)
 
+    def test_create_particles_with_id(self):
```

Comment:
@roigcarlo I have already made a small PR with this test, also with a test for the `create_bonds_with_id` function :+1: 
See #187